NativeScript/runtime/Helpers.mm

```diff
@@ -691,3 +691,13 @@
     Log(@"%s", stack.c_str());
     assert(false);
 }
+
+void tns::StopExecutionAndLogStackTraceIfInDebug(v8::Isolate* isolate) {
+    Assert(false, isolate);
+}
+
+void tns::LogDebugMessage(std::string message) {
+    if (RuntimeConfig.IsDebug) {
+        Log(@"%s", message.c_str());
+    }
+}
```

Comment:
remove and use `Log` directly

Reply:
to clarify. This function will induce people to make the mistake of building a big heavy string and discarding it here.